doc/user_guide/tools.rst

```diff
@@ -895,8 +895,39 @@ method that operates *in place*.
 Rebinning
 ^^^^^^^^^
 
-The :py:meth:`~.signal.BaseSignal.rebin` method rebins data in place down to a size
-determined by the user.
+There are two options to rebin data:
+The first uses the :py:meth:`~.signal.BaseSignal.rebin` method to rebin data,
+to a new dimensional size specified by the user. Please note the new shape must
+be a divisor of the original shape.
+
+.. code-block:: python
+
+    >>> s = hs.datasets.example_signals.EDS_SEM_Spectrum()
+    >>> print(s)
+    >>> print(s.rebin([512]))
+    <EDSSEMSpectrum, title: EDS SEM Signal1D, dimensions: (|1024)>
```

Comment:
This needs to be reordered (same as the docstrings), since I think this will fail with the new docstring/userguide tests.
```Python
>>> print(s)
<EDSSEMSpectrum, title: EDS SEM Signal1D, dimensions: (|1024)>
>>> print(s.rebin([512]))
<EDSSEMSpectrum, title: EDS SEM Signal1D, dimensions: (|512)>
```